### ISSUE TITLE
Add custom hotkeys for fast building placement

### ISSUE THREAD

**Issue by Noordfrees** [widelands/widelands#4833]
https://www.widelands.org/forum/topic/5213

Another possibility would be that one clicks on the building spot and is able to press a hotkey combination which places the building.... Those hotkeys could also be configurable.

 Maybe this feature would be even better if it would be enough to mouse over the building spot.

**Comment by frankystone** [comment]
Sorry for being late on this discussion, but i think this new feature has some flaws:

1. This option is somehow hidden for a player because its in "Options→Edit keyboard ... actions→Fastplace"
2. widelands was always known as a "slow-paced" game, but this option makes it more like a "Who knows the tools to speed up gaming will likely win the game" game
3. It lacks handling for people who don't like shortcuts
4. It is not much explained
